src/components/Card.tsx

```diff
@@ -0,0 +1,7 @@
+export const Card = (props) => {
```

Comment:
```suggestion
export const Card = ({children}) => {
```
Use destructuring whenever not unreasonable!

Reply:
Also, instead of `export const Card`, consider `const Card` and then use `export default Card;` to leverage default exports! This applies throughout the PR